proto/v1/release_service.proto

```diff
@@ -122,14 +129,43 @@ message UndeleteReleaseRequest {
   ];
 }
 
+message CheckReleaseRequest {
+  // Format: projects/{project}
+  string parent = 1 [
+    (google.api.field_behavior) = REQUIRED,
+    (google.api.resource_reference) = {type: "bytebase.com/Project"}
+  ];
+
+  // The release to check.
+  Release release = 2 [(google.api.field_behavior) = REQUIRED];
+
+  // The targets to dry-run the release.
+  // Can be database or databaseGroup.
+  // Format:
+  // projects/{project}/databaseGroups/{databaseGroup}
+  // instances/{instance}/databases/{database}
+  repeated string targets = 3;
+}
+
+message CheckReleaseResponse {
+  message CheckResult {
+    // The file id that is being checked.
+    string file = 1;
+
+    repeated Advice advices = 2;
```

Comment:
Nit: One day I want to find a better name than "advices" to be natural with popular linter tools. But I will not block you at current timestamp.